packages/web-components/src/checkbox/checkbox.styles.ts

```diff
@@ -159,17 +181,17 @@ export const checkboxStyles: (context: ElementDefinitionContext, definition: Che
         .control:active {
           background: ${SystemColors.HighlightText};
         }
-        :host(.checked) .checked-indicator {
+        :host(.checked) slot[name='checked-indicator'] {
```

Comment:
I had Kham take a pass on high contrast. I'm going to skip this one in case he already simplified it, and also per my comment about using slot names above.